src/material-design-lite.scss

```diff
@@ -25,6 +25,7 @@
 @import "typography/typography";
 
 // Components
+@import "list/_list";
```

Comment:
Components should be ordered alphabetically unless told otherwise. A few are not done this way, since they are moved around for specificity reasons.